### ISSUE TITLE
Look at measures other than testing accuracy

### ISSUE THREAD

**Issue by mackncheesiest** [mackncheesiest/ECE523_FakeNewsClassifier#1]

**Comment by mackncheesiest** [comment]
Matthew Correlation Coefficient, etc.

**Comment by mackncheesiest** [comment]
Test results for various setups
![image](https://cloud.githubusercontent.com/assets/8922497/25575440/56279b14-2e0c-11e7-86f4-8a4df8d5bbee.png)

**Comment by mackncheesiest** [comment]
The best performing classifier in terms of both Matthews Correlation Coefficient and Testing Accuracy is the prepended-hostnames-with-only-5-iterations test. It would appear that the 35 iterations llda began to overfit the training data and thus lost generalization performance. It might be good to explore what the optimal number of iterations are, but these training runs take a while.
